pom.xml

```diff
@@ -28,7 +28,7 @@
     <name>Keycloak</name>
     <description>
         Keycloak SSO
-    </description>
+</description>
```

Comment:
Some rando whitespace change here, seems to be misaligned.

Reply:
Fixed this one, which was caused by "format on save".  The rest of the whitespace issues are arguably improvements so I left them alone.